build.ps1

```diff
@@ -191,5 +195,5 @@ Write-Host "Installing cake modules using the --bootstrap argument"
 
 # Start Cake
 Write-Host "Running build script..."
-Invoke-Expression "& `"$CAKE_EXE`" `"$Script`" -target=`"$Target`" -configuration=`"$Configuration`" -verbosity=`"$Verbosity`" $UseMono $UseDryRun $UseExperimental $ScriptArgs"
+Invoke-Expression "& `"$CAKE_EXE`" `"$Script`" -target=`"$Target`" -configuration=`"$Configuration`" -verbosity=`"$Verbosity`" -AzureKeyVaultUrl=`"$AzureKeyVaultUrl`" -AzureKeyVaultAppId=`"$AzureKeyVaultAppId`" -AzureKeyVaultAppSecret=`"$AzureKeyVaultAppSecret`" -AzureKeyvaultCertificateName=`"$AzureKeyvaultCertificateName`" $UseMono $UseDryRun $UseExperimental $ScriptArgs"
```

Comment:
I notice the capitalisation for `Keyvault` is different here to the other arguments, and to [what's specified in build.cake](https://github.com/OctopusDeploy/Sashimi.AzureWebApp/pull/28/files#diff-c6eace5a70f0c2d920dfa97e988d4cad3d0131dd2b029bb2646c96222e866cf8R25). Does that matter?

Reply:
Good spot - This is what it is in Calamari at the moment, so I've missed this.

As it's not stated in the cake documentation, I've gone through the cake repository, and it looks like they're doing a case insensitive match on arguments. For example: https://github.com/cake-build/cake/blob/6dd9edec635c99d90ad853d2a4296422f38de70e/src/Cake/CakeArguments.cs

I feel like the capitalisation should be identical regardless - I'm happy to do a clean up PR to address this if you feel like this is worth doing.